src/pages/lessons/[slug].tsx

```diff
@@ -228,10 +168,39 @@ const Lesson: FunctionComponent<LessonProps> = ({initialLesson}) => {
     summary,
   } = lesson
 
-  console.log(lesson)
+  const currentState = playerState.value
+
+  console.log(`The current player state: ${currentState}`)
+
+  React.useEffect(() => {
+    switch (currentState) {
+      case 'loading':
+        if (!isEmpty(data.lesson)) {
+          send('LOADED')
+        }
+        break
+      case 'loaded':
+        if (hls_url || dash_url) {
```

Comment:
if we have urls we can display the video player

note: we are missing the "ask user for email" state here that would check to see how many free lessons they have watched and push them to create an account to continue.